src/client/src/pages/Tasks/Tasks.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from "react";
+import Header from "../../components/Header/Header.jsx";
+import AddTaskForm from "../../components/AddTaskForm/AddTaskForm.jsx";
+import TasksList from "../../components/TasksList/TasksList.jsx";
+import EmptyListNote from "../../components/EmptyListNote/EmptyListNote.jsx";
+import ActionBar from "../../components/ActionBar/ActionBar.jsx";
+import RemoveAllBtn from "../../components/RemoveAllButton/RemoveAllBtn.jsx";
+import { Toast } from "monday-ui-react-core";
+import "./tasks.css";
+
+const Tasks = ({ tasks, setTasks, taskService }) => {
+  const [editTask, setEditTask] = useState(null);
+  const [searchInput, setSearchInput] = useState("");
+  const [statusFilter, setStatusFilter] = useState("all");
+  const [presentedTasksNum, setPresentedTasksNum] = useState(0);
+  const [isErrorShown, setIsErrorShown] = useState(false);
+  const [errorMessage, setErrorMessage] = useState("");
+
+  const handleCloseError = () => {
+    setIsErrorShown(false);
+  };
+
+  const handleRemoveAll = () => {
+    setTasks([]);
+    taskService.removeAllTasks();
+  };
+
+  const showRemoveAllBtn = () => {
+    if (presentedTasksNum === tasks.length && tasks.length > 0) {
+      return <RemoveAllBtn handleRemoveAll={handleRemoveAll} />;
+    } else return null;
+  };
+
+  return (
+    <div className="container">
+      <div className="app-wrapper">
+        {isErrorShown && (
```

Comment:
the "isErrorShown" should be passed as a prop to the "open" prop of the toast component.
This component should always be mounted to the DOM in order to show the animations.
Currently once isErrorShown is changed to false you removing the toast from the dom and the effect is not shown.